src/main/java/commands/ListCommand.java

```diff
@@ -76,6 +76,9 @@ private String listModules(Access access) {
         }
         result.append(String.format(MESSAGE_EXIST, MODULE));
         for (Module m : allModules) {
+            if (m.getModuleName().equals("history")) {
```

Comment:
instead of checking for history here, you can check in loadModule in storage instead. so that it wont add the history folder as a module object